src/vcpkg/vcpkgcmdarguments.cpp

```diff
@@ -9,6 +9,61 @@
 #include <vcpkg/metrics.h>
 #include <vcpkg/vcpkgcmdarguments.h>
 
+namespace
+{
+    using namespace vcpkg;
+
+    // returns a list of known CI environment variables and the ID for their respective vendors
+    static View<std::pair<StringLiteral, StringLiteral>> get_known_ci_variables()
```

Comment:
I think this would be better as just the static constexpr array used directly rather than the indirection through a get function.

Reply:
(Or maybe even a builtin array so you wouldn't have to fill out the length)